modules/mailer/mailer.go

```diff
@@ -29,24 +30,32 @@ type Message struct {
 }
 
 // NewMessageFrom creates new mail message object with custom From header.
-func NewMessageFrom(to []string, from, subject, htmlBody string) *Message {
-	log.Trace("NewMessageFrom (htmlBody):\n%s", htmlBody)
+func NewMessageFrom(to []string, from, subject, body string) *Message {
+	log.Trace("NewMessageFrom (body):\n%s", body)
 
 	msg := gomail.NewMessage()
 	msg.SetHeader("From", from)
 	msg.SetHeader("To", to...)
 	msg.SetHeader("Subject", subject)
 	msg.SetDateHeader("Date", time.Now())
 
-	body, err := html2text.FromString(htmlBody)
+	var plainBody string
+	var err error
+	if strings.Contains(body[:100], "<html>") {
```

Comment:
Why are we checking this? What if the body starts with `<div>` (as is customary for templates) ?